experimental/segmenter/src/line.rs

```diff
@@ -230,15 +229,15 @@ pub type LineBreakIteratorUtf16<'l, 's> = LineBreakIterator<'l, 's, LineBreakTyp
 ///
 /// let breakpoints: Vec<usize> =
 ///     segmenter.segment_latin1(b"Hello World").collect();
-/// assert_eq!(&breakpoints, &[6, 11]);
+/// assert_eq!(&breakpoints, &[0, 6, 11]);
 /// ```
 ///
 /// Separate mandatory breaks from the break opportunities:
 ///
 /// ```rust
-/// # use icu::properties::{maps, LineBreak};
-/// # use icu_segmenter::LineSegmenter;
-/// #
```

Comment:
Nit: Manish suggested this setup to be hidden in https://github.com/unicode-org/icu4x/pull/3269/files/6734fa7dbdf05e31cac018987f69f2c6d288536a..618cbae9b5965b5dd61ddb56437042cd32f864b2#r1160070659. Maybe leave it as is?

Reply:
I added back at least the imports because it's very unclear reading the example where `LineBreak` comes from; it actually comes from an entirely different crate, which is not self-evident without the imports.

Reply:
I'd hide `LineSegmenter`, as this is its doc string. Maybe fully qualify the `icu_properties` items? I think that will make the example easier to parse than keeping the imports in the back of your head.

Reply:
Agreed. My comment there was _not_ about hiding the imports; it was about hiding  the setup steps for `segmenter`: all of these examples have the same preamble and can be pared down a bit to only include _new_ imports and lines of code

Reply:
+1 to hiding `LineSegmenter`, especially since it is only used in hidden code.